scripts/viz/viz_validation.r

```diff
@@ -13,7 +13,7 @@
 # 2) Scatterplot of predictions by quantified protein
 # 3) Boxplot of predictions for NF1 WT/Inactive protein
 
-set.seed(123)
+set.seed(1234)
```

Comment:
Is this seed just for geom_jitter? If so, I'd place it closer to that function and add a comment about what this is for.